src/BinDeps.jl

```diff
@@ -122,9 +118,9 @@ if Sys.KERNEL == :FreeBSD
 end
 
 if Compat.Sys.iswindows()
-    const exe7z = joinpath(_HOME, "7z.exe")
```

Comment:
it's possible other packages may have been referring to this constant

Reply:
I really hope not since this would only existing on windows. I could undo this without causing problems for us, but I'm pretty sure creating system images on windows won't work correctly.

Reply:
> creating system images on windows won't work correctly

Why's that? What is this fixing anyway?

Reply:
The issue is that `JULIA_HOME` (or I guess now `Sys.BINDIR`) doesn't exist at system image build time, which will cause functions that depend on the compiled constant to fail and runtime. Since I don't use windows I'm not particularly bothered by leaving this in the windows if block, but this could cause problems for windows user who want to make system images.